nltk/tag/stanford.py

```diff
@@ -204,6 +205,75 @@ def parse_output(self, text, sentences):
 
         raise NotImplementedError
 
+class CoreNLPTagger(CoreNLPParser, TaggerI):
+    def __init__(self, tagtype, url='http://localhost:9000', encoding='utf8'):
+        """
+        An abstract interface to POS/NER taggers of CoreNLP that returns the
+        POS/NER tags from the Stanford CoreNLP API at nltk.parse.corenlp.
+        """
+        self.tagtype = tagtype
+        super(CoreNLPTagger, self).__init__(url, encoding)
+
+    def tag_sents(self, sentences):
+        sentences = (' '.join(words) for words in sentences)
```

Comment:
Is there a way to avoid joining by space?

I would not retokenize the input, if it's already given as a list of words.